config.go

```diff
@@ -6,8 +6,8 @@ package coraza
 import (
 	"io/fs"
 
-	"github.com/corazawaf/coraza/v3/auditlog"
 	"github.com/corazawaf/coraza/v3/debuglog"
+	"github.com/corazawaf/coraza/v3/internal/auditlog"
```

Comment:
I think we need to remove the uses from this file (we also need to remove WithRules, could be separate PR)

Reply:
Very good point.